pkg/importer/dataStream.go

```diff
@@ -905,14 +909,18 @@ func (d *DataStream) copy(dest string) error {
 			return err
 		}
 	} else {
-		if util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
+		if d.VolumeMode != "Block" && util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
```

Comment:
Sometimes you compare to "Block" and sometimes to string(v1.PersistentVolumeBlock)
Can we make it consistent?

Reply:
done